From 7c97d663a69e928e2441661b14277c7d412fadab Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Fri, 24 Mar 2023 11:50:48 +0100
Subject: [PATCH 2/2] ENH: Add missing includes for standard Qt components

This is required after Slicer/Slicer#6909 to build SlicerIGT using a
Slicer install tree. Private `ui_*.h` files were included in publicly
available widgets `.h` files. Indirectly, `ui_*.h` files were including
some standard Qt components needed to build this extension. This commit
makes explicit inclusion of the needed Qt headers needed.
---
 PathExplorer/qSlicerPathExplorerModuleWidget.cxx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/PathExplorer/qSlicerPathExplorerModuleWidget.cxx b/PathExplorer/qSlicerPathExplorerModuleWidget.cxx
index 88b33a6..74bf09b 100644
--- a/PathExplorer/qSlicerPathExplorerModuleWidget.cxx
+++ b/PathExplorer/qSlicerPathExplorerModuleWidget.cxx
@@ -18,6 +18,7 @@
 // Qt includes
 #include <QDebug>
 #include <QShortcut>
+#include <QTableWidget>
 
 // SlicerQt includes
 #include "qSlicerPathExplorerModuleWidget.h"
-- 
2.39.2

